apps/docs/content/guides/guides/components.mdx

```diff
@@ -0,0 +1,170 @@
+
+### 🥇 Direct styling adjustments
+
+For straightforward customization needs, you can make adjustments directly by passing style-related props, adding CSS classes, or setting up refs for access to component elements. This approach allows you to override specific styles while keeping the component's behavior and overall styling intact.
+
+<Expand title="Details">
+#### Styled System Props
+
+Adjust properties such as `backgroundColor`, `width`, or `padding` directly on the component.
+
+```tsx
+<Button backgroundColor="primary" UNSAFE_width="148px" paddingTop="stack-xs">Click me</Button>
+```
+
+#### CSS Classes
+
+Append custom CSS classes to modify styles.
+
+```tsx
+<Button className="custom-class">Click me</Button>
+```
+</Expand>
+
+<Expand title="Example: Creating an AI Card">
+```tsx
+export function AICard({ children, className, ...rest }: CardProps) {
+  const classNames = clsx(className, "ai-gradiant-border");
+   return (
+       <Card className={classNames} {...rest}>
+           {children}
+       </Card>
+   );
+}
+```
+</Expand>
+
+### 🥇 Behavior modifications
+
+To customize a component's behavior, use refs, callbacks and controlled properties. This approach is useful for interactive features and for modifying default component behavior. This approach is ideal when you need to adjust how a component responds to user interactions without altering its visual style or structure. Read more about Controlled vs Uncontrolled components [here](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components).
+
+<Expand title="Details">
+#### Refs for Direct Access
+Use refs to access and interact with elements within a component.
+
+```tsx
+<Autocomplete wrapperProps={{ ref: myRef }}>
+    <Item key="earth">Earth</Item>
+</Autocomplete>
+```
+
+#### Event Callbacks
+Add event handlers for more control, ensuring events continue propagating as expected.
+
+```tsx
+<Button onFocus={handleFocus}>Click me</Button>
+```
+
+#### Controlled Properties
+You can control the component's behavior by passing a controlled property.
+
+```tsx
+<PopoverTrigger {...rest} open={isHovered}>
+```
+</Expand>
+
+
+<Expand title="Example: Creating a popover that opens on hover.">
+```tsx
+function MyCustomButtonPopoverTrigger({ children, ...rest }: PopoverTriggerProps) {
+   const ref = useRef();
+   const isHovered = useHover(ref);
+   return (
+       <PopoverTrigger {...rest} open={isHovered}>
+           <Button ref={ref}>Hoverable Button</Button>
+           {children}
+       </PopoverTrigger>
+   );
+}
+```
+</Expand>
+
+### 🥈 Copying component
```

Comment:
Taken from chat GPT:
Use Title Case: Capitalize the main words, usually the first and last words, and all major words in between (nouns, pronouns, verbs, adjectives, adverbs). Articles, prepositions, and conjunctions with fewer than four letters are not usually capitalized unless they start the title. For example:

"Progressive Enhancement in Disclosure Components"

I know it's not the same for french.